llvm/lib/Target/PowerPC/PPCTargetMachine.cpp

```diff
@@ -457,7 +457,7 @@ TargetPassConfig *PPCTargetMachine::createPassConfig(PassManagerBase &PM) {
 void PPCPassConfig::addIRPasses() {
   if (TM->getOptLevel() != CodeGenOptLevel::None)
     addPass(createPPCBoolRetToIntPass());
-  addPass(createAtomicExpandPass());
+  addPass(createAtomicExpandLegacyPass());
```

Comment:
We should probably just add this to the default pipeline (not in this change)

Reply:
Will create a patch for this one this gets merged